extract/extract-handlers.ts

```diff
@@ -77,6 +82,18 @@ export function extractHandlers(path: string) {
 						handlers.queue[fullDefinition.name] = fullDefinition;
 					}
 					break;
+				case HandlerTypes.Event:
+					{
+						const { definition } = handler;
+						const fullDefinition: FullHandlerDefinition<EventHandlerDefinition> =
+							{
+								...definition,
+								name: pascalCase(`Event ${definition.name}`),
```

Comment:
Why do we pre-pend these with "Event" and "queue" if we're letting people set their names for these resources, shouldn't it be on them to name them exactly as they want?

Reply:
Yes... I wanted to stay consistent with what is already happening above. I don't know why we do this besides maybe preventing naming collisions? `DeletePost` could be a name for both an event and queue handler.

@mckalexee you might want to weigh in here.